### ISSUE TITLE
Changes that would look good 

### ISSUE THREAD

**Issue by tomaswarynyca** [metroline/metroline-ui#6]
**Home**
On the home page they should cover a little more of the timeline.
A custom scroll should be added
![image](https://user-images.githubusercontent.com/41587659/108781134-314a9a80-7548-11eb-9575-50e0dec0ce18.png)

**Pipelines**
The lettering on the console should be a little smaller, it is solved by approving  #5 
A custom scroll should be added
![image](https://user-images.githubusercontent.com/41587659/108781288-6656ed00-7548-11eb-8717-ba7fe0c6bc3d.png)

**Comment by tomaswarynyca** [comment]
Another action to take into account is that it would be good to show only the first line of commit, in my case we do squash and merge which generates a commit with data that does not need to be visible in the deploy.

This photo is taken from the demo but it is for context.
![image](https://user-images.githubusercontent.com/41587659/109586787-af78e500-7ae4-11eb-88e0-5836307e5fad.png)

**EDIT:** Now with the latest update you can see what I mentioned above
![image](https://user-images.githubusercontent.com/41587659/109667723-319df380-7b4f-11eb-91ef-d40f79a5e4af.png)
![image](https://user-images.githubusercontent.com/41587659/109667761-395d9800-7b4f-11eb-8c5a-5a66dd15f6bf.png)

**Comment by gempain** [comment]
Agree. Could be solved by doing a split on newlines and only taking the first one.

Thanks so much for taking the time to suggest and document these things, it's hugely appreciated !
